### PR TITLE
Bonnefon.glogg version 1.1.4

### PR DESCRIPTION
**Pull request by EnessenE**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92626)

### CODE REVIEW

**Comment by EnessenE** [comment]
@microsoft-github-policy-service agree

File: manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Bonnefon.glogg
+PackageVersion: 1.1.4
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://glogg.bonnefon.org/files/glogg-v1.1.4-x86_64-setup.exe
+  InstallerSha256: 08FE13B713327BEF93298B6D11717A93BBAA9D49165995BE93F4A3282E76B22F
```
Comment (mdanish-kh) [inline manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml:10]:
```suggestion
  InstallerSha256: 08FE13B713327BEF93298B6D11717A93BBAA9D49165995BE93F4A3282E76B22F
  AppsAndFeaturesEntries:
    - DisplayVersion: v1.1.4-x86_64
      Publisher:
      ProductCode: glogg
```
For 1:1 match with Registry values that helps with better upgrade matching

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — DISMISSED** [review]

File: manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+PackageIdentifier: Bonnefon.glogg
+PackageVersion: 1.1.4
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: http://glogg.bonnefon.org/files/glogg-v1.1.4-x86_64-setup.exe
+  InstallerSha256: 08FE13B713327BEF93298B6D11717A93BBAA9D49165995BE93F4A3282E76B22F
+  AppsAndFeaturesEntries:
+    - DisplayVersion: v1.1.4-x86_64
+      Publisher: Bonnefon
```
Comment (mdanish-kh) [inline manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml:13]:
```suggestion
      Publisher:
```
The Publisher field written to registry/ARP table is `null`. For AppsAndFeaturesEntries, you have to provide 1:1 match with Registry values
image

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

File: manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml (hunk 2 above)
Comment (EnessenE) [reply manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml:13]: Ah I see! Apologies, misinterpreted the meaning of the field.

Comment (mdanish-kh) [reply manifests/b/Bonnefon/glogg/1.1.4/Bonnefon.glogg.installer.yaml]: No worries :)

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**